src/app/api/getPosts.ts

```diff
@@ -1,65 +1,56 @@
 import { DateTime } from 'luxon';
 import { Octokit } from 'octokit';
-import slugify from 'slugify';
 
-function toSlug(str: string) {
-  return slugify(str).toLowerCase();
-}
+import markdownToHtml from '@/lib/markdownToHtml';
 
 export type Post = {
   slug: string,
-  date: string,
+  date: DateTime,
   title: string,
   summary: string,
+  reading_time: string,
   tags: string[],
   content: string,
+  author: {
+    name?: string,
+    avatar?: string,
+  },
 };
 
 export async function getPosts(): Promise<Post[]> {
   const octokit = new Octokit();
-  const issues = await octokit.rest.issues.listForRepo({
+  const issues = (await octokit.rest.issues.listForRepo({
     owner: 'maffin-io',
     repo: 'maffin-blog',
     creator: 'argaen',
     headers: {
       accept: 'application/vnd.github+json',
     },
-  });
+  })).data.filter((issue) => !issue.pull_request);
```

Comment:
issues and PRs are returned in this endpoint. This is the way to filter according to Octokit API reference